src/all.js

```diff
@@ -2,6 +2,7 @@ import AccountMenu from './components/account-menu/account-menu';
 import TimeoutDialog from './components/timeout-dialog/timeout-dialog';
 import UserResearchBanner from './components/user-research-banner/user-research-banner';
 import CharacterCount from './components/character-count/character-count';
+import { nodeListForEach } from './common';
 
 function initAll() {
```

Comment:
Is there a way we could write a test for this?

Reply:
For `nodeListForEach`?

Reply:
No the initAll function

Reply:
Yeah, that's fair.  It would be good to add those.